scl-extension/src/main/resources/xsd/SCL_CoMPAS.xsd

```diff
@@ -222,19 +223,20 @@ SPDX-License-Identifier: Apache-2.0
     <xs:complexType name="tCompasBay">
         <xs:annotation>
             <xs:documentation xml:lang="en">
-                Define the specific Compas attributes linked to the Bay. The Main and Second Labels are label of the bay specific to each user. MainLabel and Version identify uniquely the instanciated Bay as the UUID
+                Define the specific Compas attributes linked to the Bay. The Main and Second Labels are label of the bay specific to each user.
+                MainLabel and Version identify uniquely the instanciated Bay as the UUID
                 The BayCodif is specific to each user to identify the bay kind.
             </xs:documentation>
         </xs:annotation>
         <xs:attribute name="UUID" type="xs:string" use="optional"/>
         <xs:attribute name="Indice" type="xs:string" use="optional"/>
         <xs:attribute name="Version" type="xs:string" use="optional"/>
-        <xs:attribute name="MainShortLabel" type="xs:string" use="optional"/>
+        <xs:attribute name="MainShortLabel" type="xs:string" use="required"/>
```

Comment:
What is ment by required? Should every CoMPAS project has this attribute? Or only if tCompasBay is used?

Reply:
It is required only when Private tCompasBay is used (present in file).

Reply:
It is solved: it only applies if tCompasBay is used.